webrtc.html

```diff
@@ -8852,6 +8854,34 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+	  <li>If the selector is a MediaStreamTrack, return the following objects:
+	    <ul>
+	      <li>
+		The RTCMediaStreamStats corresponding to the selector.
```

Comment:
@alvestrand Right, in my mind, we'd start with the RTCRTPStreamStats for the relevant track, which would work with a recursive "down" algorithm, as well as include at least the objects you wanted. Hmm, this tells me I was really thinking of the sender or receiver as the selector, so perhaps we should just make that explicit?

Btw, What does [`ssrcId`](https://w3c.github.io/webrtc-stats/#dom-rtcmediastreamtrackstats-ssrcids) point to? It's not documented.